### PR TITLE
Friendslist Direct Messaging

### PR DESCRIPTION
**Pull request by corneliusrenken**

### CODE REVIEW

File: client/dist/sidebar.css [hunk 1]
```diff
@@ -99,6 +99,8 @@
 }
 
 .global-chat-input {
+  margin: 0;
```
Comment (markmiw) [inline client/dist/sidebar.css]: Duplicate?

**Review by markmiw — APPROVED** [review]
Looks good to me.

**Comment by markmiw** [comment]
bussin'

File: client/dist/sidebar.css (hunk 1 above)
Comment (corneliusrenken) [reply client/dist/sidebar.css]: fixed!

**Comment by BungaloBuce** [comment]
As Mark said.... Pretty bussin'

**Review by BungaloBuce — APPROVED** [review]
Good Job!

**Pull request merged**